js/noms/src/spec-test.js

```diff
@@ -70,8 +71,24 @@ suite('Spec', () => {
 
   test('database spec', async () => {
     const invalid = [
-      'mem:stuff', 'mem::', 'mem:', 'http:', 'https:', 'random:', 'random:random',
-      'local', './local', 'ldb', 'ldb:', 'ldb:local',
+      'mem:stuff',
```

Comment:
I found it easier to port side-by-side, and easier to read regardless, with these on their own line.